Documentation/Contributors/DracoModuleManagement/README.md

```diff
@@ -0,0 +1,16 @@
+## Updating the Draco JavaScript Decoder Module
+
+We use a custom build of the [Draco](https://github.com/google/draco) decoder JavaScript module to allow for IE11 compatibility.
+
+
+1. Download and install a make tool.
+    * Preferably, use Linux or [the Linux Subsystem for Windows 10](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and `make` to build.
+    * Optionally, [MSYS2](http://www.msys2.org/). Follow installation instructions and add the path to MSYS2's usr/bin to your `PATH`. For example, if the path is `C:\msys64\usr\bin`, run `PATH=%PATH%;C:\msys64\usr\bin`.
+2. Clone or download the [Draco release](https://github.com/google/draco/releases).
```

Comment:
Actually, one comment.  If you clone Draco from git, you'll get master and not an official release, right?  We should probably always include an official release in Cesium